### PR TITLE
feat: add archive command

### PR DESCRIPTION
**Pull request by abtris**
New command list all yours API project in Apiary and save them to local file in folder where you execute command. 

```
apiary archive 
```

File ended as `apib` - don't detect if is Swagger or API Blueprint

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +69,19 @@ Commands:
...
 
+#### archive
+
+```
+$ apiary help archive
+Usage:
+  apiary archive
+
+Options:
+  [--output=FILE]  # Write API Description Document into specified file
```
Comment (opichals) [inline README.md:80]: If I read the code correctly this doesn't seem to be supported.

Comment (abtris) [reply README.md:80]: I will try remove this option.

Comment (abtris) [reply README.md:80]: Do you think that make sense make some output path option there?

Comment (abtris) [reply README.md]: I removed this option now.

File: README.md [hunk 2]
```diff
@@ -57,7 +57,7 @@ export APIARY_API_KEY=<your_token>
 ```
 $ apiary help
 Commands:
-  apiary archive                      # Archive All Your API Description Documents from apiary.io
+  apiary archive                      # Archive All Your API Description Documents from apiary.io. Going to be written to files named as API project subdomain.
```
Comment (opichals) [inline README.md:60]:
```suggestion
  apiary archive                      # Archive All Your API Description Documents from apiary.io to local files named following [api-project-subdomain.apib] pattern.
```

Comment (abtris) [reply README.md]: ok, this is better, updating.

**Review by opichals — APPROVED** [review]
Looks good.

**Pull request merged**